app/routes.py

```diff
@@ -1 +1,229 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, make_response, request, abort, jsonify 
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+#to call slack api in this file:
+from dotenv import load_dotenv
+import requests 
+import os
+load_dotenv()
+
+# Blueprints:
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message": f"{cls.__name__} {model_id} invalid"}, 400))
+    
+    model = cls.query.get(model_id)
+
+    if not model:
+        abort(make_response({"message": f"{cls.__name__} {model_id} not found"}, 404))
+
+    return model
+
+
+@tasks_bp.route("", methods=["POST"])
+def add_task():
+    request_body = request.get_json()
+    if not "title" in request_body or not "description" in request_body:
```

Comment:
```suggestion
    if "title" not in request_body or "description" not in request_body:
```
small style note again - having `<Object> not in ...` is the preferred syntax over `not <Object> in ...`